testfiles/DOAToPTAC.idf

```diff
@@ -1910,13 +1910,13 @@
   Sizing:Zone,
     SPACE1-1,                !- Zone or ZoneList Name
     SupplyAirTemperature,    !- Zone Cooling Design Supply Air Temperature Input Method
-    13.8,                    !- Zone Cooling Design Supply Air Temperature {C}
+    15.5,                    !- Zone Cooling Design Supply Air Temperature {C}
```

Comment:
Because of changes in the amount of outdoor air (primary air), there were warning error messages that are related to capacity to flow ratio out of range.